run.sh

```diff
@@ -138,4 +138,5 @@ podman run \
     -e CRYOSTAT_HIBERNATE_DIALECT="$CRYOSTAT_HIBERNATE_DIALECT" \
     -e CRYOSTAT_HBM2DDL="$CRYOSTAT_HBM2DDL" \
     -e CRYOSTAT_LOG_DB_QUERIES="true" \
+    -e CRYOSTAT_DEV_MODE="$CRYOSTAT_DEV_MODE" \
```

Comment:
What does the dev mode do and how do we use it?

Reply:
`CRYOSTAT_DEV_MODE=true CRYOSTAT_AUTH_MANAGER=io.cryostat.net.NoopAuthManager sh smoketest.sh`, then in your browser open `https://localhost:8181/api/v2.2/graphiql/`.

`sh devserver.sh` also sets the `CRYOSTAT_DEV_MODE=true`.

Currently the GraphiQL endpoint is the only thing gated behind the dev mode variable.